docs/contributing.rst

```diff
@@ -1,4 +1,197 @@
 Contributing to AllTheBacteria
 ==============================
 
-To do
+We welcome contributions to AllTheBacteria. Please read on for instructions
+on how to contribute to the project.
+
+Background information
+----------------------
+
+Where does everything live?
+
+* Assemblies, metadata and all analysis files are hosted at the
+  AllTheBacteria project on OSF: https://osf.io/xv7q9/.
+  ie this is where we expect users of the data to go.
+* Reproducibility files (scripts, methods descriptions) are on the
+  AllTheBacteria GitHub repository: https://github.com/AllTheBacteria/AllTheBacteria.
+  This is where contributors should add their reproducibility files, and where
+  anyone should look for methods as opposed to the actual data.
+  The exception is large files (eg containers), which go on OSF.
+* Documentation for the end-user (ie how can I get and use the data?) is here
+  on readthedocs.
+
+Before starting
+^^^^^^^^^^^^^^^
+
+Before starting any work that you intend to contribute, please check if it has
+already been done (it's already on the  OSF AllTheBacteria project) or is
+already on the to do/in progress list
+(see the `AllTheBacteria GitHub planning project <https://github.com/orgs/AllTheBacteria/projects/1/views/1>`_).
+
+Every task on the project page should link to a GitHub issue. If the work
+you are interested in is already in progress and you want to comment and/or
+help, then please reply to the relevant existing GitHub issue.
+
+To contribute, you will need:
+
+* A `GitHub <https://github.com>`_ account
+* An `OSF <https://osf.io>`_ account
+
+
+Reproducibility
+---------------
+
+Please consider reproducibility before running any analysis.
+
+Methods and scripts should be ultimately added to the public
+AllTheBacteria GitHub repository. At a minimum, this means anyone should be
+able to figure out exactly what you ran (tool versions, commands etc) and have
+access to any scripts that were used. If not already publicly available,
+we will insist that any containers used are added to OSF.
+
+We understand that everyone works differently. It is perfectly reasonable to
+make a script/pipeline that you used to run on your own compute cluster,
+knowing that it would likely break if someone else tried running it elsewhere.
+We just require that all scripts are made available, so that the process is
+clearly documented, even though it is not necessarily easy to rerun.
+For a concrete example see the
+`checkm2 analysis <https://github.com/AllTheBacteria/AllTheBacteria/tree/main/reproducibility/All-samples/checkm2>`_,
+which was run on the EBI cluster using SLURM.
+
+
+
+SOP for contributing
```

Comment:
"Standard operating procedure (SOP)"

Reply:
going to do that in the text in the next sentence. Want to keep the heading short